step.yml

```diff
@@ -124,6 +124,16 @@ inputs:
       value_options:
       - "yes"
       - "no"
+  - fetch_tags: "yes"
+    opts:
+      category: Debug
+      title: Fetch tags
+      summary: Adds --no-tags to fetch commands
```

Comment:
This could be confusing because setting the flag excludes `--no-tags` instead of adding it. Maybe we could call this `no_tags` instead or clarify the summary a little bit more? `Disabling this flag adds --no-tags to fetch commands`
@godrei which solution should we go with?

Reply:
I agree it should be the reverse and have the setting disabled by default